### PR TITLE
Tamil language added

### PR DESCRIPTION
**Pull request by thearasu**
Locale added for Indian Tamil language.

### CODE REVIEW

**Comment by thearasu** [comment]
I just added a new language, please approve!

File: common/config/src/locale/ta-IN.ts [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+export type LocaleOptions = DeepPartial<LocaleConfig>
+export type LocaleNames = Exclude<keyof LocaleConfig, 'locale' | 'wordSpace'>
+
+export function defineLocale(locale: LocaleOptions) {
+  return locale
+}
+
+export function defineLocaleConfig(locale: LocaleConfig) {
+  return locale
+}
```
Comment (qmhc) [inline common/config/src/locale/ta-IN.ts:184]: These are redundant codes, please remove.

**Review by qmhc — CHANGES_REQUESTED** [review]
Thanks for your contribution.

**Review by thearasu — COMMENTED** [review]
done

File: common/config/src/locale/ta-IN.ts (hunk 1 above)
Comment (thearasu) [reply common/config/src/locale/ta-IN.ts]: removed it

File: common/config/src/locale/ta-IN.ts [hunk 2]
```diff
@@ -0,0 +1,173 @@
+import type { DeepPartial } from '../types'
+
+export interface LocaleConfig {
```
Comment (qmhc) [inline common/config/src/locale/ta-IN.ts:3]: Well, I missed here, you can refer the `en-US.ts` to fix here.

```suggestion
import { defineLocaleConfig } from './helper'

export function taINLocale() {
  return defineLocaleConfig({
```

**Review by qmhc — CHANGES_REQUESTED** [review]

**Review by thearasu — COMMENTED** [review]
Now this is fine, I believe.

**Comment by qmhc** [comment]
Yes, you run `pnpm run lint` and I will merge it. 👍

**Review by qmhc — APPROVED** [review]

**Comment by thearasu** [comment]
Could you merge it?

**Pull request merged**